Source/driver/Castro_setup.cpp

```diff
@@ -467,6 +467,14 @@ Castro::variableSetUp ()
 #endif
 #endif
 
+#ifdef NSE_NET
+  // Adding descriptor for chemical potentials used for NSE_NET
+  store_in_checkpoint = true;
+  desc_lst.addDescriptor(Chemical_Pot_Type, IndexType::TheCellType(),
+			 StateDescriptor::Point, 1, NCHEM,
+			 interp, state_data_extrap, store_in_checkpoint);
```

Comment:
I am not completely sure we only want `1` ghost cell.  In the Strang burner, we call the burner on `ngrow_state` ghost cells, so I think that we need this to have that same amount.